src/cf-buttons/usage.md

```diff
@@ -1,11 +1,20 @@
-- [Default button](#default)
-- [Secondary button](#secondary)
-- [Destructive action button](#destructive)
-- [Disabled button](#disabled)
-- [Super button](#super)
+- [Atoms](#atoms)
+  - [Default button](#default-button)
+  - [Secondary button](#secondary-button)
+  - [Destructive action button](#destructive-action-button)
+  - [Disabled button](#disabled-button)
+  - [Super button](#super-button)
+  - [Button links](#button-links)
+  - [Icon buttons](#icon-buttons)
```

Comment:
Ah, sorry, I just dove right into the code 😁 

I commented, though, because the Less file was in `/molecules`, so that's the inconsistency I saw.

Incidentally, this is the example I was going to raise in the call about atomic design. Perhaps we should hold off and discuss it with everyone at the workshop.